dev/com.ibm.ws.jdbc_fat_krb5/fat/src/com/ibm/ws/jdbc/fat/krb5/containers/PostgresKerberosContainer.java

```diff
@@ -10,42 +10,32 @@
  *******************************************************************************/
 package com.ibm.ws.jdbc.fat.krb5.containers;
 
-import java.io.FileInputStream;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
-import java.util.Properties;
 
 import org.testcontainers.DockerClientFactory;
 import org.testcontainers.containers.Network;
 import org.testcontainers.containers.PostgreSQLContainer;
-import org.testcontainers.containers.output.OutputFrame;
 import org.testcontainers.utility.DockerImageName;
 
 import com.ibm.websphere.simplicity.log.Log;
-import com.ibm.ws.jdbc.fat.krb5.FATSuite;
+
+import componenttest.containers.SimpleLogConsumer;
 
 public class PostgresKerberosContainer extends PostgreSQLContainer<PostgresKerberosContainer> {
 
     private static final Class<?> c = PostgresKerberosContainer.class;
-    private static final Path reuseCache = Paths.get("..", "..", "cache", "postgres.properties");
+
     // NOTE: If this is ever updated, don't forget to push to docker hub, but DO NOT overwrite existing versions
     private static final String IMAGE = "aguibert/krb5-postgresql:1.0";
     private static final DockerImageName postgresImage = DockerImageName.parse(IMAGE)
                     .asCompatibleSubstituteFor("postgres");
```

Comment:
Similar to previous comment.  Consider externalizing these two values (via public) so they can be used instead of hardcoded values in other classes.